parseComCode/main.js

```diff
@@ -30,6 +30,141 @@ Parse.Cloud.job("populateSearch", function(request, status) {
   });
 });
 
+//A background job to populate usageCounts for languages and tags
```

Comment:
Would you please add comments describing how to invoke it and/or how to schedule it to run regularly.